arches/app/models/system_settings.py

```diff
@@ -123,7 +123,11 @@ def setup_node(node, parent_node=None):
 
         # set any values saved in the instance of the Arches System Settings Graph
         for tile in models.TileModel.objects.filter(
-            resourceinstance__graph_id=self.SYSTEM_SETTINGS_RESOURCE_MODEL_ID
+            resourceinstance__graph_id=self.SYSTEM_SETTINGS_RESOURCE_MODEL_ID,
+            # Get tiles only from the latest graph publication.
+            resourceinstance__graph_publication=models.F(
+                "resourceinstance__graph__publication"
+            ),
```

Comment:
It would be profoundly silly, but if you version the system settings graph and delete the map settings, this would fail. But testing that silly case is at least a clean test case for...